### ISSUE TITLE
Timezone support for current weather response

### ISSUE THREAD

**Issue by vdobrikov** [xSAVIKx/openweathermap-java-api#15]
Here is the announcement:
https://openweather.co.uk/blog/post/we-have-integrated-time-zones-our-weather-api-products

And diff:
```diff
diff --git a/api-model/src/main/java/org/openweathermap/api/model/currentweather/CurrentWeather.java b/api-model/src/main/java/org/openweathermap/api/model/currentweather/CurrentWeather.java
index 4468b40..a893495 100644
--- a/api-model/src/main/java/org/openweathermap/api/model/currentweather/CurrentWeather.java
+++ b/api-model/src/main/java/org/openweathermap/api/model/currentweather/CurrentWeather.java
@@ -28,4 +28,5 @@ public class CurrentWeather extends AbstractWeatherInformation {
     private String visibility;
     @SerializedName("sys")
     private SystemParameters systemParameters;
+    private long timezone; // Shift in seconds from UTC
 }
```

**Comment by xSAVIKx** [comment]
@vdobrikov thx for the issue! Looks like this change should be actually added to other results as well.

**Comment by vdobrikov** [comment]
I'd glad to help, but for some reason I wasn't able to push my branch into this repo.

**Comment by clintfoster** [comment]
This would be very useful since otherwise there is no way to determine whether it is day or night based on comparing "now" in the weather TZ with the returned sunrise/sunset times.

**Comment by xSAVIKx** [comment]
Fixed in #18 and released in v1.4.0.
